src/components/calcite-notice/calcite-notice.tsx

```diff
@@ -139,13 +145,13 @@ export class CalciteNotice {
   /** close the notice emit the `calciteNoticeClose` event - <calcite-notice> listens for this */
```

Comment:
@macandcheese does calcite-notice actually listen for this? I can't find any references to this event...

Reply:
I think that's a remnant... We could probably remove those methods altogether and just rely on "active" being set. Notice has none of the "queue" complications that Alert does so that'd be an easy one.

Reply:
OK, I think leaving the events is good for consuming devs for "events up props down". But in the future we could just remove this mention in the docs

Reply:
Yerp - leave events, remove "open" / "close" methods.